src/ScheduleFunctions.cpp

```diff
@@ -353,29 +358,28 @@ Stmt build_provide_loop_nest(const map<string, Function> &env,
 
     // Default schedule/values if there is no specialization
     Stmt stmt = build_provide_loop_nest_helper(
-        func_name, prefix, start_fuse, dims, site, values,
-        def.split_predicate(), f_sched, def.schedule(), is_update);
+            func_name, prefix, start_fuse, dims, site, values,
```

Comment:
Pointless reindenting? I think the rest of the compiler uses 4 spaces when an arg list wraps.

Reply:
My editor must have done that.